src/i18n.js

```diff
@@ -114,6 +130,10 @@ const resources = {
         'As informações da sua WiFi não são enviadas ao servidor. Nenhum dado é coletado por esse website. Veja o',
       'desc.source': 'código fonte',
       'wifi.login': 'Autenticação WiFi ',
+      'wifi.size': 'Tamanho',
+      'wifi.size.small': 'Pequeno',
+      'wifi.size.medium': 'Médio',
+      'wifi.size.large': 'Grande',
```

Comment:
@pedrorenan is it grammatically correct?

Reply:
Yes!

Reply:
Thanks!